### ISSUE TITLE
error while running

### ISSUE THREAD

**Issue by NotAditya01** [Uo1428/ALL-IN-ONE-Discord-Bot-#30]
node:internal/modules/cjs/loader:1183
  return process.dlopen(module, path.toNamespacedPath(filename));
                 ^

Error: The module '\\?\C:\Users\Aditya\Desktop\ALL-IN-ONE-Discord-Bot--main\node_modules\canvas\build\Release\canvas.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 108. This version of Node.js requires
NODE_MODULE_VERSION 102. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).
    at Object.Module._extensions..node (node:internal/modules/cjs/loader:1183:18)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (C:\Users\Aditya\Desktop\ALL-IN-ONE-Discord-Bot--main\node_modules\canvas\lib\bindings.js:3:18)
    at Module._compile (node:internal/modules/cjs/loader:1099:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12) {
  code: 'ERR_DLOPEN_FAILED'

**Comment by Uo1428** [comment]
update nodejs version.
using replit?

**Comment by vxry3** [comment]
> update nodejs version. using replit?

no, because im not paying 20 a month just to make it private, so dont use replit since people can just find your repil and find the token and steal the bot,

**Comment by Uo1428** [comment]
you may use [`NVM`](https://nvm.sh). and use node v18.12.0

**Comment by vxry3** [comment]
> you may use [`NVM`](https://nvm.sh). and use node v18.12.0

sheluvsbyron is my discord add me cus i need help running the bot
